ga4gh/datamodel/reads.py

```diff
@@ -75,3 +76,80 @@ def toProtocolElement(self):
         readGroup.referenceSetId = None
         readGroup.sampleId = None
         return readGroup
+
+    def getReadAlignments(self, referenceName, referenceId, start, end):
+        """
+        Returns an iterator over the specified reads
+        """
+        # TODO use an index if it is found (accompanying .bai file)
```

Comment:
Someone correct me if I am wrong, but I don't think SAM files can use indexes.  Do we not want to support SAM files or BAM files without indexes?